samples/app-complete-sample/csharp/README.md

```diff
@@ -9,9 +9,10 @@ extensions:
   contentType: samples
   technologies:
   - Tabs
-  - Microsoft Bot Framework
+  - Microsoft Bot Framework v4
   createdDate: 9/22/2017 5:54:09 PM
-description: "Sample that shows how to build a bot for Microsoft Teams in C#."
+  updateDate:  
+description: "Sample that shows how to build a bot for Microsoft Teams in C# with bot framework v4."
 ---
 
 # Microsoft Teams Bot in C#
```

Comment:
Could you please update the Git clone command? currently it's pointing to old repo. 

`git clone https://github.com/OfficeDev/microsoft-teams-template-bot-CSharp.git`

Reply:
Updated git clone command